pkg/endpointmanager/manager.go

```diff
@@ -460,6 +463,28 @@ func (mgr *EndpointManager) AddEndpoint(owner regeneration.Owner, ep *endpoint.E
 	return nil
 }
 
+func (mgr *EndpointManager) AddHostEndpoint(ctx context.Context, owner regeneration.Owner,
+	proxy endpoint.EndpointProxy, allocator cache.IdentityAllocator, reason string, nodeName string) error {
+	ep, err := endpoint.CreateHostEndpoint(owner, proxy, allocator)
+	if err != nil {
+		return err
+	}
+
+	if err := mgr.AddEndpoint(owner, ep, reason); err != nil {
+		return err
+	}
+
+	// Give the endpoint a security identity
+	newCtx, cancel := context.WithTimeout(ctx, launchTime)
+	defer cancel()
+	ep.UpdateLabels(newCtx, labels.LabelHost, nil, true)
```

Comment:
Is `labels.LabelHost` defined anywhere?

Reply:
Yes, in `pkg/labels/labels.go`, next to `labels.LabelHealth`.